LEVEL2/2884.py

```diff
@@ -0,0 +1,14 @@
+h, m = map(int, input().split())
+
+if m > 44:
```

Comment:
```suggestion
if m >= 45:
```
이렇게 바꾸는 것이 더 맞는 의미 입니다.